go.mod

```diff
@@ -3,15 +3,14 @@ module github.com/docker/cli-docs-tool
 go 1.18
 
 require (
-	github.com/spf13/cobra v1.2.1
+	github.com/spf13/cobra v1.5.0
```

Comment:
Still a bit on the fence on updating here if we don't require features from the newer version (may be good to keep it at "lowest working version")